int32 frame_rate_value
---
int32 frame_rate_new_value
